src/Solver/FreeSurfaceIntegrator.cpp

```diff
@@ -233,10 +233,15 @@ void seissol::solver::FreeSurfaceIntegrator::initializeSurfaceLTSTree(  seissol:
 {
   seissol::initializers::LayerMask ghostMask(Ghost);
 
+  auto const is_duplicate = [&ghostMask, ltsLut](unsigned ltsId) {
+    return ltsId != ltsLut->ltsId(ghostMask, ltsLut->meshId(ghostMask, ltsId));
+  };
+
```

Comment:
1) Name should be formatted like isDuplicate
2) Probably should be called isUnique, as it is only used in the negated form